api/time_bucket_ng.md

```diff
@@ -32,24 +32,21 @@ SELECT timescaledb_experimental.time_bucket_ng('1 year', date '2021-08-01');
 (1 row)
 ```
 
-To split time into buckets we use a starting point in time called `origin`. The
-default origin is '2000-01-01'. `time_bucket_ng` can't work with timestamps
-earlier than `origin`:
+To split time into buckets, `time_bucket_ng()` uses a starting point in time
+called `origin`. The default origin is `2000-01-01`. `time_bucket_ng` cannot use
+timestamps earlier than `origin`:
```

Comment:
I think some explanation is required here. I am not sure why the example below does not work . @afiskon  Could you add some text here that describes why the example fails?

Reply:
The reason is that the timestamp is earlier than the default origin, as stated in the sentence. Any ideas on how to clarify this?

Reply:
I think part of the confusion is that we do not document what the argument names are. I wasn't sure what date meant here.  See: https://docs.timescale.com/api/latest/hyperfunctions/time_bucket/#time-bucket. 

This api section defintely needs a required + optional arguments section.
@afiskon We should also revise the error message in the source code and use the correct argument names.